src/structure/action/ban.ts

```diff
@@ -82,7 +82,8 @@ export class BanAction extends Action {
 					humanizeDuration(this.duration, { largest: 3, round: true })
 				)
 				.addField('Expires', prettyDate(this.expiresAt))
-				.setTimestamp();
+				.setTimestamp()
+				.setFooter('forums.minehut.com', 'https://i.imgur.com/dnyXrKy.png');
```

Comment:
This isn't the Minehut logo my dude, I'm also not sure how much I like the logo being there at all.
Also just putting the link to the forums doesn't really indicate that you're meant to appeal there, maybe there could be something like `appeal at forums.minehut.com/whatever/here`

Reply:
That'd be the old Minehut logo.
If we keep the logo there, I recommend we use something more permanent than Imgur. Imgur deletes images after a while.

Reply:
There isn't a Minehut logo in any other bot cmds so I don't think it should be there at all (especially since it isn't very future proof in case Minehut changes its logo), but yes I agree with you.

Reply:
> That'd be the old Minehut logo.
> If we keep the logo there, I recommend we use something more permanent than Imgur. Imgur deletes images after a while.

Can you cite a source? What is the retention period? I believe Imgur items last forever.

Reply:
It turns out I was wrong! That's my bad, sorry :p